src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml.cs

```diff
@@ -261,22 +262,44 @@ public void SelectFilter(ItemFilter selectedFilter)
 
         private void TabsPackageManagement_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            TabItem previousTabItem = e.RemovedItems.Count > 0 ? e.RemovedItems[0] as TabItem : null;
-            TabItem selectedTabItem = e.AddedItems.Count > 0 ? e.AddedItems[0] as TabItem : null;
+            NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
```

Comment:
Is this taking a long time?
I'm looking for insight into why it was made async.

Reply:
Nope, I'm currently reverting it... was an experiment

Reply:
Sorry, I somehow cherry-picked an experimental commit here. This file is no longer relevant (removed usings)